### ISSUE TITLE
POM dissappears at close proximity when starting recording

### ISSUE THREAD

**Issue by Cineminecraft** [daipenger/minema#6]
my POM looks fine using shaders, also when I start my path using bauercam. But when I press F4 to record the POM suddenly disappears around me at close proximity, you can see here it's still there at the far end of the block, but not close to me
![unknown](https://user-images.githubusercontent.com/13148927/57407138-c4370b00-71e2-11e9-96b4-abaa4d05dfed.png)

**Comment by daipenger** [comment]
Ressource pack?
Shader pack?
Version of Minema, Forge and OptiFine?
Any other mods?

**Comment by Cineminecraft** [comment]
Resourcepack: Stratum 1K
Shader: Continuum 2.0.2

Mods:
Forge 1.12.2-14.23.5.2768
Minema 1.12.2
Bauercam 1.13 for 1.12.2
Optifine 1.12.2_HD_U_E3
Worldedit forge 1.12-6.1.8-dist

**Comment by daipenger** [comment]
Alright, I will try to reproduce it. It's hard to find these logical bugs as nothing is crashing but maybe there is a fix or workaround.

**Comment by Cineminecraft** [comment]
thank you, if you need any more details or settings I am using don't hesitate to ask.

**Comment by daipenger** [comment]
Looks fine to me:
https://youtu.be/wNc0UqhlMkM

Same version of Forge and OptiFine.
Continuum 2.0.2 with default settings
Minema v3.4.2 with default settings (you didn't specify which one you are using but v3.4.2 is the latest)
Stratum 1K is not public, so I used S&K Photo Realism 512 with a fitting POM pack (https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/resource-packs/2690082-resource-pack-shader-support-parallax-pom-bump) for this test. It may make a difference since it's only a 512x512 pack? But I doubt it.

But what I did notice: POM cuts off at the near plane. But that is normal and happens regardless of recording.

Strange, I trust you, but something is going on on your side that's not there on my side.
